pkg/grafanaclient/test.go

```diff
@@ -0,0 +1,85 @@
+package main
```

Comment:
Please remove it.
If you need some kind of test please do it with standard test package https://golang.org/pkg/testing/ and build tags https://golang.org/pkg/go/build/#hdr-Build_Constraints, as far as such tests require Grafana instance and can't be called unit, but rather integration.